docs/authoring.md

```diff
@@ -259,3 +225,85 @@ There are some template parameters that have a special meaning:
 |Name|Description|
 |---|---|
 |`server`| It is used to let the template know which server from the AsyncAPI specification file you want to use. In some cases, this may be required. For instance, when generating code that connects to a specific server. Use this parameter in case your template relies on users' information about what server from the specification file they want to use during generation. You also need this parameter if you want to use `server.protocol` notation within `conditionalFiles` configuration option. Once you decide to specify this parameter for your template, it is recommended you make it a mandatory parameter otherwise feature like `conditionalFiles` is not going to work if your users do not use this parameter obligatory.
+
+
+## React
+[React](https://reactjs.org) is the newest render engine which we strongly suggest you should use for any new templates. It is only the default render engine to enable backward compatibility.
+* It enables the possiblity of [debugging](#debugging-react-template) your template (this is not possible with Nunjucks).
+* It provides better stack traces once something is not working.
+* Provides better support for separating code into more manageable chunks.
+* The readability of the template are much better then Nunjucks syntax.
```

Comment:
I think we should keep it because it outlines the issues that is inherit by Nunjucks that they will encounter if they go for that instead. People can make their own decision, but I think it is a good idea to have it 😄